### PR TITLE
remove default nic from vm

### PR DESCRIPTION
**Pull request by mrturkmen06**

### CODE REVIEW

File: svcs/ctfd/interception.go [hunk 1]
```diff
@@ -305,11 +305,6 @@ func (ri *registerInterception) Intercept(next http.Handler) http.Handler {
 
 	updateRequest := func(r *http.Request, t *store.Team) error {
 		var err error
-		name := r.FormValue("name")
```
Comment (jbjjbjjbj) [inline svcs/ctfd/interception.go]: Why are these changes included?

Comment (mrturkmen06) [reply svcs/ctfd/interception.go]: It caused problems regarding to teams, "Unknown token" error started to appear, in addition for unfinished events, daemon could not assign all teams to events.
